src/helper/util.ts

```diff
@@ -251,6 +251,52 @@ export class ZkWasmUtil {
     return bns;
   }
 
+  static BNToHexString(bn: BN): string {
+    return "0x" + bn.toString("hex");
+  }
+
+  static BytesToHexStrings(data: Uint8Array, chunksize: number = 32): string[] {
+    let hexStrings = [];
+    let bns = this.bytesToBN(data, chunksize);
+    for (let i = 0; i < bns.length; i++) {
+      hexStrings.push(this.BNToHexString(bns[i]));
+    }
+    return hexStrings;
+  }
+
+  static BNToBytes(bn: BN, chunksize: number = 32): Uint8Array {
+    return new Uint8Array(bn.toArray("le", chunksize));
+  }
+
+  static HexStringToBN(hexString: string, chunksize: number): BN {
+    // Should begin with 0x
+    this.validateHex(hexString);
+
+    let bn = new BN(hexString.slice(2), 16);
+
+    // Check if the BN is more than expected chunksize bytes
+    if (bn.byteLength() > chunksize) {
```

Comment:
move it to the BNToBytes, so HexStringToBN no need chunksize input, just like BNToHexString